profile/README.md

```diff
@@ -2,6 +2,6 @@
 
 Welcome to Backbase's open source organisation. Regardless to if you are a customer, partner or employee you can create and share with everyone in the Backbase Community via this organisation.
 
-You'll find repo's here directly maintained by or Backbase teams and essestial tools use as part of our developer experaince. See [learn.backbase.com](https://learn.backbase.com) to pick up a course on our best practices and [developer.backbase.com](https://developer.backbase.com/) to see the offical developer documentaion.
+You'll find repo's here directly maintained by or Backbase teams and essestial tools use as part of our developer experaince. See [backbase.io/learning](https://backbase.io/learning) to pick up a course on our best practices and [backbase.io/developers](https://backbase.io/developers/) to see the offical developer documentaion.
```

Comment:
Please fix also the typos `essestial` `experaince`, `documentaion`.
First sentence is not proper English grammar, yes?

Reply:
Thanks Jasper, would you mind updating the PR?